pkg/core/app/app.go

```diff
@@ -70,6 +71,16 @@ func (a *App) Setup(_ context.Context) error {
 	if utils.IsDockerKind(a.kind) && isDetachMode(a.logger, a.cmd, a.kind) {
 		return fmt.Errorf("application could not be started in detached mode")
 	}
+	var composeFilePath string
```

Comment:
This is not the right way to do it, we will not parse the docker compose file path using a flag. Rather we will parse the appCmd and then get the filename.